### ISSUE TITLE
Need to create universal example for yii-app.

### ISSUE THREAD

**Issue by githubjeka** [dmstr/yii2-adminlte-asset#33]
@schmunk42 
basic example is same as advanced example, except
```php
backend\assets\AppAsset::register($this); //advanced
app\assets\AppAsset::register($this); //basic
```

it can be replace to 

```php
if (class_exists('backend\assets\AppAsset')) {
    backend\assets\AppAsset::register($this);
	} else {
	  app\assets\AppAsset::register($this);
	  }
```

Now basic use vesion 1.x admin-lte. I try to used for basic app
```
'view' => [
			 'theme' => [
				 'pathMap' => [
					'@app/views' => '@vendor/dmstr/yii2-adminlte-asset/example-views/yiisoft/yii2-advanced-app'
				 ],
			 ],
		],
```
```json
 "dmstr/yii2-adminlte-asset": "2.*"
```
It works.

Therefore we can delete basic and advanced expamples and create a universal.
But how to be with backward compatible? Need a new version :)

**Comment by schmunk42** [comment]
They are just examples anyway and there's no stable 2.0 in this repo yet, so go ahead ;)

**Comment by githubjeka** [comment]
Please update `Releases` and ReadMe. Thanks.

**Comment by schmunk42** [comment]
So, we can release a stable 2.0?

**Comment by githubjeka** [comment]
Yes

**Comment by githubjeka** [comment]
If is phundamend ready. :)

**Comment by schmunk42** [comment]
Just released 2.0.0 - thanks for you work.
